source/style-guide/style/placeholder-variable-text.txt

```diff
@@ -1,147 +1,182 @@
+          :copyable: false
+
+          .. code-block:: console
+
+             mongod --port <portNumber> \
+                    --dbpath <tempDatabasePath> \
+                    --setParameter ttlMonitorEnabled=false
 
-     - :samp:`nova boot {serverName} --image {image} --flavor {flavor}
-       --nic net-id=net1_id`
+       .. code-block:: console
+          :copyable: false
+
+          mongod --port <portNumber> \
+                 --dbpath <tempDatabasePath> \
+                 --setParameter ttlMonitorEnabled=false
 
    * - Use lowercase letters except when showing a multiple-word
        placeholder.
 
-       To show a multiple-word placeholder, don't separate the words
-       with spaces or symbols. To distinguish the words in the
-       placeholder, capitalize the first letter of each word after the
-       first word (called camelCase). Don't capitalize the first word.
+       To show a multiple-word placeholder:
+
+       - Don't separate the words with spaces or symbols
+       - Capitalize the first letter of each word after the first word
+         (called :wikipedia:`camelCase </Camel_case>`)
+       - Never capitalize the first word
 
        .. note::
-          Use lowercase and camelCase unless you have to follow the
-          conventions of the programming language. For example, you
-          might need to use underscores (account_ID) or all capitals
-          (ACCOUNT_ID).
+          Use lowercase and camelCase unless you need to follow the
+          conventions of the programming language.
+
+     - *password*, *eventTypeName*, *apiKey*, *hostId*
 
-     - *password* *serverName* *apiKey* *tenantId*
+   * - Use one or more whole words to represent a placeholder.
 
-   * - In general, use one or more whole words to represent a
-       placeholder. Don't sacrifice clarity for brevity. Create
-       placeholders that are descriptive and meaningful.
+       - Don't sacrifice clarity for brevity.
+       - Create descriptive and meaningful placeholders.
 
-     - *device* (instead of *dev*)
+     - *device*; not *dev*
 
-       *installationDirectory* (instead of *installDir*)
+       *installationDirectory*; not *installDir*
 
-       *mode* (instead of *########*)
+       *mode*; not *########*
 
-When explaining a placeholder, use the following guidelines.
+Placeholder Explanation Guidelines
+----------------------------------
+
+When explaining a placeholder, use the following guidelines:
 
 .. list-table::
    :widths: 40 60
    :header-rows: 1
 
-   * - Guidelines
+   * - Guideline
 
      - Example
 
-   * - Avoid stand-alone clauses that begin with *where*. Instead, use
-       a sentence.
-
-     - *Use:*
-
-       **https://dfw.bigdata.api.MongoDBcloud.com/v1.0/yourAccountId/**
-
-       *yourAccountId* is your actual account number, which is returned
-       as part of the authentication service response.
-
-       *Avoid:*
+   * - Use a sentence to describe the placeholder parameter.
```

Comment:
Question: do we have any guidelines for describing placeholders in code comments? I see instances of that in some of our existing documentation (though I'm not an advocate of it).

Reply:
Not as such. Maybe a separate ticket to discuss?